packages/apps/desktop-viewer-test/public/links.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "textKey": "home.links.viewerComponent",
+    "url":"https://www.npmjs.com/package/@itwin/desktop-viewer-react"
+  },
+  {
+    "textKey": "home.links.customizeViewer",
+    "url":"https://developer.bentley.com/tutorials/itwin-viewer-hello-world/"    
+  }
+]
```

Comment:
is it possible to open the sign in redirect using the same popups as you do for these?

Reply:
Havent been able to find a way so far...core does not seem to offer an option, but perhaps a hack is possible